[0m16:35:13.133919 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020D3788E650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020D36A0E110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020D36A0FD60>]}


============================== 16:35:13.141737 | a6e2acc9-d246-4f82-8703-e243e14afb89 ==============================
[0m16:35:13.141737 [info ] [MainThread]: Running with dbt=1.9.4
[0m16:35:13.142856 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\Katia\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m16:35:28.206205 [debug] [MainThread]: Starter project path: C:\Users\Katia\Desktop\other projects\dbt-schemify\dbt-schemify\myvenv\lib\site-packages\dbt\include\starter_project
[0m16:35:28.242992 [info ] [MainThread]: 
Your new dbt project "test_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:35:28.246440 [info ] [MainThread]: Setting up your profile.
[0m16:35:28.256503 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m16:35:28.258503 [debug] [MainThread]: Command `dbt init` failed at 16:35:28.258503 after 15.35 seconds
[0m16:35:28.259503 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020D3788E650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020D39968C70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020D39968C40>]}
[0m16:35:28.260504 [debug] [MainThread]: Flushing usage events
[0m16:35:28.743226 [debug] [MainThread]: An error was encountered while trying to flush usage events
